test-jest/cartesian/Bar.spec.tsx

```diff
@@ -1,7 +1,8 @@
+import { render } from '@testing-library/react';
+import * as _ from 'lodash';
```

Comment:
```suggestion
import _ from 'lodash';
```
should be sufficient, right?